packages/bits/spec/components/paginator/paginator.e2e.ts

```diff
@@ -1,27 +1,29 @@
-import { by, element } from "protractor";
-
 import { Atom } from "../../atom";
 import { Helpers } from "../../helpers";
 import { PaginatorAtom } from "../public_api";
 
 describe("USERCONTROL paginator", () => {
     let atom: PaginatorAtom;
```

Comment:
Maybe this? (to be more specific now that we have other paginators on the page)

```suggestion
    let basicPaginator: PaginatorAtom;
```

Reply:
Agree, this is more clear.